apps/ios/SimpleXChat/AppGroup.swift

```diff
@@ -276,6 +276,68 @@ public class IntDefault: Default<Int> {
     }
 }
 
+public class StringDefault: Default<String> {
+    public func get() -> String {
+        self.defaults.string(forKey: self.key)!
+    }
+}
+
+public class GenericDefault<T> {
```

Comment:
some hammer factory factory factory...
If it's used just for one type, better define it in place, without any abstractions.